ext/web/02_event.js

```diff
@@ -1037,8 +1037,12 @@ class EventTarget {
 
     const { listeners } = self[eventTargetData];
     if (callback !== null && ReflectHas(listeners, type)) {
-      listeners[type] = ArrayPrototypeFilter(
-        listeners[type],
+      listeners[type].capture = ArrayPrototypeFilter(
+        listeners[type].capture,
```

Comment:
This appears to be wrong, I think you should use a logic similar to this:
```
((typeof listener.options === "boolean" &&
          listener.options === options.capture) ||
          (typeof listener.options === "object" &&
            listener.options.capture === options.capture)) &&
        listener.callback === callback
```

Reply:
I think the original suggestion is correct.